typings/index.d.ts

```diff
@@ -3661,6 +3674,30 @@ declare module 'discord.js' {
     $device?: string;
   }
 
+  interface WidgetMember {
+    id: string;
+    username: string;
+    discriminator: string;
+    avatar: null;
+    status: PresenceStatus;
+    deaf?: boolean;
+    mute?: boolean;
+    self_deaf?: boolean;
+    self_mute?: boolean;
+    suppress?: boolean;
+    channel_id?: string;
+    avatar_url: string;
+    activity?: {
+      name: string;
+    };
+  }
+
+  interface WidgetChannel {
```

Comment:
This interface here is a regular channel but the channel has the possibility to not be already cached by the client (and possibly can't be cached)